components/Animations/src/Builders/AnimationBuilder.Default.cs

```diff
@@ -92,7 +92,11 @@ public sealed partial class AnimationBuilder
         RepeatOption? repeat = null,
         EasingType easingType = DefaultEasingType,
         EasingMode easingMode = DefaultEasingMode,
+#if !HAS_UNO
         FrameworkLayer layer = FrameworkLayer.Composition)
+#else
+        FrameworkLayer layer = FrameworkLayer.Xaml)
+#endif
```

Comment:
@Sergio0694 I looked for everywhere we reference this enum to change the default to Xaml when targeting Uno, there shouldn't be anything else I need to do, eh?

I think the only case that didn't follow this similar last default parameter pattern was this one here for the XAML Property:

https://github.com/CommunityToolkit/Windows/pull/121/files#diff-cae1f3eb16f43b8b50f14e417e321525ea038d5cadb76ad33c0035c7b89f650aR33

Reply:
Yup, the methods here and that property in `CustomAnimation<TValue, TKeyFrame>` I think should do it 👍